### ISSUE TITLE
Remove Acquia Purge and Acquia connector from the project

### ISSUE THREAD

**Issue by lhridley** [mautic/mautic.org-website#13]
The Acquia Purge and Acquia Connector projects are specific to the Acquia Hosting platform.  Since this project is now hosted on DrupalSolid, these modules are no longer needed.

This issue is to track work to remove these modules from the project.

This issue will block #12 
This issue will need to be completed after #11
